cranelift/codegen/src/isa/riscv64/inst/vector.rs

```diff
@@ -253,6 +255,32 @@ impl fmt::Display for VecAluOpRRR {
     }
 }
 
+impl VecAluOpRRImm5 {
+    pub fn opcode(&self) -> u32 {
+        // Vector Opcode
+        0x57
+    }
+    pub fn funct3(&self) -> u32 {
+        // OPIVI
+        0b011
+    }
+    pub fn funct6(&self) -> u32 {
+        // See: https://github.com/riscv/riscv-v-spec/blob/master/inst-table.adoc
+        match self {
+            VecAluOpRRImm5::Vadd => 0b000000,
```

Comment:
Well, It is the same, but not all Opcodes work on both formats. `vadd` works for both, but `vsub` does not have a `vsub.vi` format. It would be nice if we could share the funct6 mapping somehow though since it is exactly the same on both.

Reply:
Ah ok makes sense, in that case sounds good to me to have a minor amount of duplication 👍